plat/kvm/Config.uk

```diff
@@ -102,6 +102,7 @@ config KVM_VMM_FIRECRACKER
 	select KVM_BOOT_PROTO_LXBOOT
 	select HAVE_MMIO
 	select VIRTIO_MMIO_LINUX_COMPAT_CMDLINE
+	select LIBUKPM_PS2 if ARCH_X86_64 && LIBUKBOOT_MAINTHREAD
```

Comment:
Should be `select LIBUKPS2`. Also `drivers/ukps2` needs a `Config.uk`.

Moreover the way to express the dependency between libukboot and this feature would be to define a `HAVE_PS2` in `drivers/ukps2/` as:
```
config HAVE_PS2
    bool

config LIBUKPS2
    bool "Minimal PS/2 driver for CtrlAltDel"
    depends on HAVE_PS2
```

Then in `KVM_VMM_FIRECRACKER`:
```
select HAVE_PS2
```

And finally in `libukboot`:
```
imply LIBUKBOOT_MAINTHREAD if HAVE_PS2
```

Reply:
Following an offline discussion, it's been decided that we proceed as such:
- in plat/kvm/Config.uk for FC
select HAVEPS2
imply LIBUKPS2_KBD
- in lib/ukboot/Config.uk
imply LIBUKBOOT_MAINTHREAD if HAVE_SHUTDOWN
- in drivers/ukps2/kbd/Config.uk
select HAVE_SHUTDOWN # hey by selecting the kbd you now have shutdown available
- in plat/Config.uk
config HAVE_SHUTDOWN
- in lib/ukps2/Config.uk
config HAVE_PS2